### DISCUSSION TITLE
Random input thread freezes

### DISCUSSION THREAD

**Discussion by uzervlad** [ppy/osu#20082] [Q&A] [answered]
It's not always as bad as shown on the video, but is still very much noticeable

https://user-images.githubusercontent.com/24654479/188007199-192e5a2b-a0dd-4586-a6e7-cc6c69f5d2e9.mp4

Possibly related logs (quit as soon as I noticed a spike):
[performance.log](https://github.com/ppy/osu/files/9473202/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9473203/runtime.log)

**Comment by peppy** [comment, marked as answer]
If you are seeing input thread spikes, this means something else on your PC is stalling the windows event loop.

Try shutting down applications until it stops, trial and error to find the cause. It will not only be affecting osu! but most other apps on your PC too.
